src/modules/Profile.rb

```diff
@@ -13,26 +13,10 @@ class ProfileClass < Module
     # All these sections are handled by AutoYaST (or Installer) itself,
     # it doesn't use any external AutoYaST client for them
     GENERIC_PROFILE_SECTIONS = [
```

Comment:
... is it because AutoYaST package itself contains these *_auto scripts, but did not provide that info in it's own .desktop file? Or because it was improperly parsed?